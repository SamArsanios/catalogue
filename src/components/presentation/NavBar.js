import React from 'react';
import { Link } from 'react-router-dom';

const NavBar = () => (
  <div>
    <nav className="navbar navbar-light bg-dark mb-5">
      <div className="container">
        <div className="navbar-header">
          <Link className="navbar-brand text-white text-lg brand-text" to="/">Movies Catalogue</Link>
        </div>
        <ul className="navbar-nav ml-auto text-light d-inline-block">
          <li className="nav-item d-inline-block mr-4">
            <Link className="navbar-brand text-white text-lg brand-text" to="/search" />
          </li>
        </ul>
      </div>
    </nav>
  </div>
);

export default NavBar;
