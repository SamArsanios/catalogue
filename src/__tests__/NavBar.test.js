import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import NavBar from '../components/presentation/NavBar';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
