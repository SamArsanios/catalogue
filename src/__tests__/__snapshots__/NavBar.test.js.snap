// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav
    className="navbar navbar-light bg-dark mb-5"
  >
    <div
      className="container"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand text-white text-lg brand-text"
          href="/"
          onClick={[Function]}
        >
          Movies Catalogue
        </a>
      </div>
      <ul
        className="navbar-nav ml-auto text-light d-inline-block"
      >
        <li
          className="nav-item d-inline-block mr-4"
        >
          <a
            className="navbar-brand text-white text-lg brand-text"
            href="/search"
            onClick={[Function]}
          />
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
